src/NuGet/VisualStudio/VS.ExternalAPIs.Roslyn.Package.csproj

```diff
@@ -130,6 +130,7 @@
       <!-- Include a few dependencies of Roslyn. These right now are consumed out of the ExternalAPIs NuGet package to do assembly consistency checking in the main VS
            repo. We should remove these and insert the packages directly, but for now we'll include these to limit the work needed to consume this package. -->
       <_File Include="$(ArtifactsBinDir)Roslyn.VisualStudio.Setup\$(Configuration)\net472\Humanizer.dll" TargetDir="" />
+      <_File Include="$(ArtifactsBinDir)Roslyn.VisualStudio.Setup\$(Configuration)\net472\Microsoft.CodeAnalysis.FlowAnalysis.Utilities.dll" TargetDir="" />
```

Comment:
@dotnet/roslyn-infrastructure for help on insertion changes for the new dependency. Will the change in this file and the next file suffice to ensure this assembly is copied next to Features/Workspaces assembly in VS install? I verified the setup project changes in subsequent files work fine and the assembly is loaded correctly in the RoslynDev hive.

Reply:
I think this file is for the roslyn external API package, so it probably not necessary to include this assembly here. 

---
In reply to: [278337122](https://github.com/dotnet/roslyn/pull/35248#discussion_r278337122) [](ancestors = 278337122)

Reply:
Sure, I will revert this change.

Reply:
@genlu - The comment about this say that the dependencies added here are needed to perform assembly dependency consistency checks in VSO repo. Are we sure not adding an entry here will not fail those dependency checks? Once you confirm, I can revert the change to this file.

Reply:
hmm, I'm not sure how the check works. But it sounds like we should add them back. @jasonmalinowski?

Reply:
Ideally if there's a new NuGet package you should be directly inserting that NuGet package. This is a bit of a hack which is a "well, we aren't, so we'll just stick the binary somewhere else".

The "consistency check" in this case is a tool that's looking at types to catch a minimum set of IVT breaks. So if you don't have a dependency of Roslyn inserted somewhere for it to look at...it won't be able to validate it. It's a suppressable warning by the tool, but better to include it if you can.